docs/built-in-makers.md

```diff
@@ -7,7 +7,7 @@ details on metric makers.
 Makers for Druid Aggregation Operations
 ---------------------------------------
 
-These makers are used to make first-level metrics, and are in a one-to-one mapping to the 
```

Comment:
Your editor may automatically changed this file (deleting padding spaces). We should avoid changing like this and keep this whole file untouched.

Reply:
Lines should not be added or deleted. Removing trailing spaces is fine, though.

Reply:
I can add those trailing spaces back if necessary.